src/getHumanAge.test.js

```diff
@@ -7,4 +7,48 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should return an array', () => {
+    expect(getHumanAge(0, 0))
+      .toBeInstanceOf(Array);
+  });
+
+  test('should return 0 for inputs 0', () => {
```

Comment:
```suggestion
  test('should return [0, 0] for inputs (0, 0)', () => {
```

because now it seems like the function returns a number, but it's not true